frontend/src/components/Profile.vue

```diff
@@ -0,0 +1,184 @@
+              @change="uploadImage($event)"
+              class="button-icon"
+            />
+            <div class="left-icon"></div>
+            <p>
+              Upload your <br />
+              profile picture
+            </p>
+          </div>
+          <div class="right">
+            <div class="container__group">
+              <input
+                type="text"
+                placeholder="First name*"
+                name="firstname"
+                id="firstname"
+                required
+              />
+            </div>
+            <div class="container__group">
+              <input
+                type="text"
+                placeholder="Last name*"
+                name="lastname"
+                id="lastname"
+                required
+              />
+            </div>
+
+            <div class="container__city">
+              <select name="City" id="city" required>
+                <option value="City" disabled selected>City*</option>
+                <option value="Brussels">Brussels</option>
+                <option value="Leuven">Leuven</option>
+                <option value="Namur">Namur</option>
+                <option value="Antwerp">Antwerp</option>
+                <option value="Liège">Liège</option>
+                <option value="Ghent">Ghent</option>
+                <option value="Brugge">Brugge</option>
+                <option value="Eupen">Eupen</option>
+                <option value="Arlon">Arlon</option>
+              </select>
+              <span class="container__select-icon"
+                ><i class="zmdi zmdi-chevron-down"></i
+              ></span>
+            </div>
+          </div>
+        </div>
+
+        <div class="container__radio">
+          <label for="usertype" class="radio-label">Status*</label>
+          <div class="container__radio-item1">
+            <input type="radio" name="status" id="newcomer" checked />
+            <label for="newcomer">Newcomer</label>
+            <span class="container__radio-check"></span>
+          </div>
+
+          <div class="container__radio-item1">
+            <input type="radio" name="status" id="volunteer" />
+            <label for="volunteer">Volunteer</label>
+            <span class="container__radio-check"></span>
+          </div>
+        </div>
+
+        <div class="container__group">
+          <input
+            type="email"
+            placeholder="E-mail*"
+            name="email"
+            id="email"
+            required
+          />
+        </div>
+        <div class="container__group">
+          <input
+            type="password"
+            placeholder="Password*"
+            name="password"
+            id="password"
+            required
+          />
+        </div>
+
+        <div class="container__group">
+          <div class="container__select">
+            <select name="Gender" id="gender" required>
+              <option value="Gender" disabled selected>Gender*</option>
```

Comment:
The gender should be seen by default as text, there shouldn't be a select here. it should transform into select when you click on Edit. Is it possible to make?

Reply:
I am thinking to disable all fields on loading profile page. When user clicks on Edit button, then only fields get enabled. Also,I am thinking to populate corresponding value in select(dropdown) field itself. So, without using textbox, hope we can implement it. Not sure. Anyway will try. If it doesn't work, I will change to textboxes. What do you think?

Reply:
Good idea – go ahead!